paddle/fluid/framework/ir/graph_helper.cc

```diff
@@ -99,6 +100,63 @@ std::vector<ir::Node *> TopologySortOperations(const Graph &graph) {
   return ret;
 }
 
+std::vector<ir::Node *> SortOperationsInSequence(const Graph &graph) {
```

Comment:
Put this in your pass.cc? We don't want to encourage others to use the op_desc from program